backend/src/bundles/email/email.service.ts

```diff
@@ -0,0 +1,27 @@
+import sgMail from '@sendgrid/mail';
+
+import { config } from '~/common/packages/packages.js';
+
+class EmailService {
+    public constructor() {
+        sgMail.setApiKey(config.ENV.SEND_GRID.MAIL_API_KEY);
```

Comment:
All variables from config provide to constructor and save to private fields

Reply:
done